__test__/creds/fixtures/CredFiltered1.json

```diff
@@ -2,14 +2,15 @@
   "id": null,
   "issuer": "jest:test",
   "issuanceDate": "2018-06-19T21:52:47.512Z",
-  "identifier": "credential-cvc:TestWithExcludes-v1",
+  "identifier": "credential-cvc:TestWithExcludes-v3",
```

Comment:
Same argument as before, should we not keep the existing fixtures to proof that legacy credentials still work / can be verified?